xml/System/Type.xml

```diff
@@ -13866,10 +13866,12 @@ If the value of this property is `true` for an array type, it can be used to cre
         <Parameter Name="position" Type="System.Int32" Index="0" FrameworkAlternate="netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="position">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="position">The typed parameter position.</param>
+        <summary>Returns a signature type object that can be passed into the <c>Type[] types</c> array of a <see cref="Overload:Type.GetMethod" /> method to represent a generic parameter reference.</summary>
+        <returns>A signature type object that can be passed into the <c>Type[] types</c> array of a <see cref="Overload:Type.GetMethod" /> method to represent a generic parameter reference.</returns>
```

Comment:
```suggestion
        <returns>A signature type object that can be passed into the <c>Type[] types</c> array of a <see cref="Overload:System.Type.GetMethod" /> method to represent a generic parameter reference.</returns>
```

Reply:
Wondering if this would read better. The current sentence kind of assumes you know the parameter name, which reads a bit odd.

```suggestion
        <returns>A signature type object that can be passed into the <c>Type[]</c> array parameter of a <see cref="Overload:System.Type.GetMethod" /> method to represent a generic parameter reference.</returns>
```

Reply:
Okay yes, good idea. Let me also change your other similar suggestions.